google-service-api-json-key.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
We should never commit keys / sensitive data like these to version control. We should store it somewhere else and only commit "safe" / public stuff here. Could you please regenerate all this private data and store it elsewhere? You can maybe do what it's done similarly with the other keys in this repo... (take a look at all the `example` files...).

Don't forget to update the documentation on how the user should procede to get this set up properly.